modules/osd-create-cluster-gcp-account.adoc

```diff
@@ -34,6 +34,16 @@ For more information about service account keys, click the information icon loca
 .. Select a cluster version from the *Version* drop-down menu.
 .. Select a cloud provider region from the *Region* drop-down menu.
 .. Select a *Single zone* or *Multi-zone* configuration.
++
+//ifdef::osd-on-gcp[]
```

Comment:
Is the commenting-out here just placeholder for future uncommenting, or could the `ifdef`/`endif` be omitted for now?